### PR TITLE
New package: Ahoy.Ahoy version 2.1.1

### PR DESCRIPTION
**Pull request by superusercode**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107670)

### CODE REVIEW

File: manifests/a/Ahoy/Ahoy/2.1.1/Ahoy.Ahoy.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Ahoy.Ahoy
+PackageVersion: 2.1.1
+InstallerType: portable
+ReleaseDate: 2023-04-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/ahoy-cli/ahoy/releases/download/v2.1.1/ahoy-bin-windows-amd64.exe
+  InstallerSha256: CB672356B394D79F792448B0B9926D2C7E1E2768AFE7D9AA389C88B155F2139E
```
Comment (Trenly) [inline manifests/a/Ahoy/Ahoy/2.1.1/Ahoy.Ahoy.installer.yaml:11]:
```suggestion
  InstallerSha256: CB672356B394D79F792448B0B9926D2C7E1E2768AFE7D9AA389C88B155F2139E
  Commands:
  - "ahoy"
```

**Comment by stephengillie** [comment]
HI @superusercode,

This package appears to have installed correctly, but when launched does not seem to be interactive nor a non-interactive server. 
image
Examining the server with Netstat shows the application isn't listening on any ports. Is this behavior expected?

**Comment by Trenly** [comment]
Assigning back to MSFT to ensure this doesn't get missed

[Policy] Needs Attention

**Comment by stephengillie** [comment]
We don't really have a way of specifying a configuration file as a dependency. Though one might be useful, as many console applications error in validation due to a missing config file.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**